requirements.txt

```diff
@@ -0,0 +1,8 @@
+pytest
+pytest-sugar
+math
+typing
+itertools
+string
+os
```

Comment:
there are a lot of built-in modules here, like os, string, itertools etc. you should not include built-in modules in requirements